src/components/biggive-tipping-slider/biggive-tipping-slider.tsx

```diff
@@ -86,13 +89,20 @@ export class BiggiveTippingSlider {
   };
 
   render() {
+    const currencySymbol = Intl.NumberFormat('en-GB', {
+      style: 'currency',
+      currency: this.donationCurrency,
+    })
+      .format(0)
+      .replace(/[0\.]/g, '');
```

Comment:
Instead of using Intl NumberFormat just to get the currency symbol, it seems like it might make more sense to pass in the donation number instead of 0 to format and re-run it inside the move handler. Would also give us the commas as thousand separators that we don't have at the moment since we're just using `Number.toString`

If we're not using the actual number then a simple switch statement might be just as good as `NumberFormat`

Reply:
@bdsl Yeah, it did feel odd. I pointed out on the Jira issue that I think the rounding also needs re-thinking / scoping properly, so this didn't feel worth spending a lot of time on right now.

I think I also assumed the class wrapping the number might be getting used to style it differently, but that seems not to be the case.

I've pushed a refactor now to make it a bit less hacky, also removing `Math`-rounding – let me know what you think?

Reply:
LGTM - ideally I'd make the parameter to `format` a single object containing the number and the currency, because that's a value object we could re-use across the system to start to enforce a rule that they always have to travel together. And then probably make the number be in minor units (e.g. pence) instead of major so that it's not a floating point number.

But I think what you've got looks good for now.